docs/getting-started/fundamentals-block-development/file-structure-of-a-block.md

```diff
@@ -0,0 +1,79 @@
+# File structure of a block
+
+At its simplest, a block in the WordPress block editor is a JSON object with a specific set of properties that is registered in the system so it can be used.
+
+The recommended way to register a block is via a plugin. With the `create-block` tool you can quickly scaffold the structure of the files required to create a plugin that registers a block. 
+
+The files generated by this tool are a good reference of the files that can be involved in the definition and registration of a block.
+
+[![Open File Structure of a Block Diagram in excalidraw](https://developer.wordpress.org/files/2023/11/file-structure-block.png)](https://excalidraw.com/#json=YYpeR-kY1ZMhFKVZxGhMi,mVZewfwNAh_oL-7bj4gmdw "Open File Structure of a Block Diagram in excalidraw")
+
+### `<plugin-file>.php`
+
+A block is usually added to the block editor using a WordPress plugin. In the main PHP file of the plugin the block is usually registered on the server side.
+
+<div class="callout callout-info">
+For more on creating a WordPress plugin see <a href="https://developer.wordpress.org/plugins/plugin-basics/">Plugin Basics</a>, and <a href="https://developer.wordpress.org/plugins/plugin-basics/header-requirements/">Plugin Header requirements</a> for explanation and additional fields you can include in your plugin header.
+</div>
+
+### `package.json`
+
+[`package.json`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json) is a configuration file for a NodeJs project. In this file you define the NPM dependencies of the block and the scripts used for local work.
+
+### `src` folder
+
+In a standard project you'll place your block files in the `src` folder. By default, the build process with `wp-scripts `will take files from this folder and will generate the bundled files in the `build` folder. 
+
+### `block.json`
+
+This files contains the metadata of the block, and it's required for its registration. Among other things it provides properties to define the files you need to be part of the definition of the block:
```

Comment:
I found this difficult to read. Does this describe block.json or the list of files below the text? I think I struggled because of the plural form.

Reply:
Again this file is not required at all :). 

It just makes things easier.

Reply:
I have updated this part taking into account the  feedback of both you 🙏

Reply:
Isn't block.json required to be able to style the block with theme.json? If only I could recall exactly where I got that information from. I assume it is related to API versions...